src/Constants.ts

```diff
@@ -77,3 +77,20 @@ export const HARDCODED_RULES = {
 		category: 'Scanner Internal'
 	}
 };
+
+export enum Severity {
+	NONE = 0,
+	LOW = 3,
+	MODERATE = 2,
+	HIGH = 1
+}
+
+export const SeverityMap = new Map([
```

Comment:
I initially had it set up this way but there was quite a lot of repetition where each engine had a couple methods doing the same thing just with different values, so we decided a map would allow the function to be in the abstract class and anything specific to an engine was stored in the map.

Reply:
Nevermind! Found a way where only the part unique to the engine can be in each respective engine.